pages/api/parse-html.ts

```diff
@@ -0,0 +1,65 @@
+import { NextApiRequest, NextApiResponse } from "next";
```

Comment:
what exactly is the reason for making this an endpoint rather than a function?

Reply:
It runs on the edge runtime. Edge functions can't import arbitrary node packages